Transfer/Sources/CreateTransferController.swift

```diff
@@ -42,7 +42,6 @@ final class CreateTransferController: UITableViewController {
         (TransferButtonCell.self, TransferButtonCell.reuseIdentifier),
         (TransferNotesCell.self, TransferNotesCell.reuseIdentifier)
     ]
-
```

Comment:
why this change?

Reply:
Empty space added. This happens when i added and removed the constant(Pagename , Pagegroup)